### DISCUSSION TITLE
Localization banners

### DISCUSSION THREAD

**Discussion by schalkneethling** [mdn/yari#3465] [Q&A]
After some idea sharing and live prototyping, we believe we have landed on something that can work for both:

1. Showing that a page is localized but, more importantly,
2. that the content is community maintained

Essentially, this is what we landed on roughly:

## Ideas

![Screenshot 2021-04-07 at 17 28 23](https://user-images.githubusercontent.com/10350960/113892953-d5ba3080-97c6-11eb-9c84-b5270aa628e1.png)

What do you think? Ideas and suggestions encouraged. Thank you.

Again, once we have settled at a place that most folks are happy, we will turn this into an issue for implementation.

**Comment by dalexsy** [comment]
Something to consider is if we need something like Wikipedia's [Omboxes](https://en.wikipedia.org/wiki/Template:Ombox) if we have more pieces of information than just localization information, such as archival status. Although this approach might be aggravating to users reading localized content and would be repeatedly exposed to the same info box over and over if we made it more prominent than shown in the examples you posted.
![image](https://user-images.githubusercontent.com/44966503/114032712-dad9b700-987c-11eb-9794-2f96e29e3846.png)

**Reply by schalkneethling** [reply]
I believe for that type of information we are using the block at the bottom of article pages

![Screenshot 2021-04-08 at 15 43 33](https://user-images.githubusercontent.com/10350960/114037339-2f7f3100-9881-11eb-9687-8112f165562c.png)

**Reply by escattone** [reply]
@dalexsy Since we're going to stop displaying all of the archived content on June 30th, and I can't think of any other immediate use case of this style of "banner", I don't think it's worth building a more general system for this kind of thing.

**Comment by escattone** [comment]
Thanks for opening this discussion @schalkneethling. I would vote for the second idea, with the yellow background and the left-justified text. As suggested by text of the example ideas, I think we'd have to provide hard-coded translations for each of the 10 supported locales (we're dropping from about 32 to the top 10 locales on April 30th). I think @chrisdavidmills should decide on the actual text and link that we show in the "banner", but I know that @ddbeck suggested that we keep it positive and proactive rather than anything that might feel like a warning.

**Reply by escattone** [reply]
One more thing. We should provide two different text strings, one for actively-maintained locales (one of `ru`, `zh-CN`, `zh-TW`, `ja`, or `fr`) , and another for locales that aren't.

**Reply by schalkneethling** [reply]
That sounds like a good idea @escattone

**Reply by ddbeck** [reply]
> @ddbeck suggested that we keep it positive and proactive rather than anything that might feel like a warning.

Yes. To elaborate a little, I wanted to avoid making readers and translators feel as if their language is non-standard, while also making it clear that the content is translated. I was thinking it might be nice to frame it as a CTA, along the lines of "This page was translated from English by the community. Learn more by joining the [\<insert language\> community on MDN]." and link to something that would connect the reader to whomever is doing the translations in their particular language.

**Comment by peterbe** [comment]
Perhaps a nit-pick, but the yellow background calls for a lot of attention on the eyes. It's the only bright-yellow thing other than possibly some warning note cards. But I guess it could be a blessing because if it gets more attention, than a subtle background would, is that it might bring more attention to people wanting to get involved and improve the translations even more.

**Comment by escattone** [comment]
Unless anyone has an objection, I think we should move ahead and create an issue from this discussion as follows:
- Let's go with @dalexsy's second mockup:
    - Located just below the breadcrumb bar and much thinner than that bar
    - Some kind of highlighted background (yellow or light yellow or whatever color @schalkneethling and @dalexsy feel is best). Would we like a different (perhaps stronger) color for locales that are **not** actively maintained? I suspect we would. Thoughts?
    - Left justified text
- Let's go with translations of @ddbeck's text for the 5 active locales (`ru`, `zh-CN`, `zh-TW`, `ja`, and `fr`): `This page was translated from English by the community. Learn more by joining the [<insert language> community on MDN].` I'm not sure what each language community's link should be, but we can figure that out in the issue.
- We need different text for the locales that aren't actively maintained. @ddbeck or @chrisdavidmills, would either of you like to define that text? Perhaps translations of something like: `This page was translated from English by the community, but is not actively maintained so is likely out-of-date. See [this](https://github.com/mdn/translated-content#promoting-an-inactive-locale-to-tier-1) if you'd like to change that.`
- I think we should hard code the translations into the banner.

**Reply by chrisdavidmills** [reply]
> I'm not sure what each language community's link should be, but we can figure that out in the issue.

We could link to individual headings on the [MDN Localization page](https://developer.mozilla.org/en-US/docs/MDN/Contribute/Localize). I just updated the page so it is not out-of-date.

> This page was translated from English by the community, but is not actively maintained so is likely out-of-date. See [this](https://github.com/mdn/translated-content#promoting-an-inactive-locale-to-tier-1) if you'd like to change that.

This text is not too bad. Ilike to see it tweaked just a little bit, to this:

> This page was translated from English by the community, but it is not actively maintained therefore it may be out-of-date. If you'd like to help maintain it, [find out how to activate inactive locales](https://github.com/mdn/translated-content#promoting-an-inactive-locale-to-tier-1).

**Comment by escattone** [comment]
FYI, I created #3588 from this discussion.
